src/marked/marked.module.scss

```diff
@@ -1,4 +1,5 @@
 @import '~terra-mixins/lib/Mixins';
+@import './clinical-lowlight-theme/marked.module';
```

Comment:
Import for orion-fusion-theme file missing.

Reply:
Must've been early when I was writing these 👀